test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -276,8 +276,7 @@ def test_execjob_end_reject_request(self):
         # Submit a multi-node job
         a = {'Resource_List.select':
              '1:ncpus=1:host=%s+1:ncpus=1:host=%s' %
-             (self.momA.shortname, self.momB.shortname),
-             'Resource_List.place': 'excl'}
+             (self.momA.shortname, self.momB.shortname)}
         j = Job(TEST_USER, attrs=a)
         j.set_sleep_time(60)
```

Comment:
can we reduce the time to 30s and the subsequent times as well to avoid running this test for more than a minute?